### ISSUE TITLE
[Analyzer Suggestion]: Simplify `ToUpper(CultureInfo.InvariantCulture)` to `ToUpperInvariant()`

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/runtime#77863]
As in the title, the analyzer finds `stringObj.ToUpper(CultureInfo.InvariantCulture)` calls and suggests simplifying them with the shorter version `stringObj.ToUpperInvariant()`

**Comment by tarekgh** [comment]
@Youssef1313 thanks for filing the issue. I see this as a low priority as both usages are legitimate. Nothing wrong to fix here more than a different writing. Do you see that the same way?

**Comment by Youssef1313** [comment]
> @Youssef1313 thanks for filing the issue. I see this as a low priority as both usages are legitimate. Nothing wrong to fix here more than a different writing. Do you see that the same way?

Yes they are equivalent, it's just simplification.

It could be more suitable as `IDExxxx` instead of `CAxxxx`, but since it's related to .NET APIs I opened it in this repo first.

**Comment by tarekgh** [comment]
I agree it is shorter but not more readable :-) Actually explicitly using `CultureInfo.InvariantCulture` is more explanatory. I'll close this one as I said it is something low priority and not helping much. Thanks again for your report.
